src/elementTree.js

```diff
@@ -283,9 +280,9 @@ function processToken(token, source) {
         token.type = 'Punctuator';
         token.sourceCode = token.value = '>';
     } else if (type === tt.jsxName) {
-        token.type = 'JSXIdentifier';
+        token.type = 'Identifier';
     } else if (type === tt.jsxText) {
-        token.type = 'JSXText';
+        token.type = 'String';
```

Comment:
I am not sure about this change.

As far as I understand, `JSXText` is this thing:
```
<tag>
  Text <-- this
</tag>
```

Right? If so, then having `String` token is incorrect, as `Strings` do always have quotes, but `JSXText` does not.